src/utils/helpers.ts

```diff
@@ -206,3 +206,5 @@ export const abbreviateNumber = (
   }
   return shortValue + suffixes[suffixNum];
 };
+
+export const isNullOrUndefined = val => val === undefined || val == null;
```

Comment:
`val == null` already checks for null and undefined.

In my opinion this function is redundant, you could just check `rateToBalance.earn != null`.

Reply:
Usage of `==` in JS is antipattern and I wouldn't use it at all.

Reply:
And I haven't noticed it's used here, good catch :D @rick23p please change `val == null` to `vall === null`.